app/controllers/stafftools/assignment_invitations_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Stafftools
+  class AssignmentInvitationsController < StafftoolsController
```

Comment:
I had that before, but rubocop didn't like it. I can add a rule to ignore that if `class Stafftools::AssignmentInvitationsController < StafftoolsController` is preferable